bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/types/DateTimeType.java

```diff
@@ -108,7 +109,7 @@ public DateTimeType(String zonedValue) {
             throw new IllegalArgumentException(zonedValue + " is not in a valid format.", invalidFormatException);
         }
 
-        zonedDateTime = date;
+        zonedDateTime = date.withFixedOffsetZone();

```

Comment:
It looks like the right thing to streamline the SPI. But in opposition I found https://github.com/eclipse/smarthome/issues/5076 which questions if `withFixedOffsetZone()` is the right way to go. Unfortunately with no result. @openhab/core-maintainers wdyt?

Reply:
withFixedOffsetZone() is applied to the other constructors so it makes sense to apply to this one too. It allows to get identical DateTimeType when calling the different constructors with the same data.